apps/statsig/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,240 @@
+    }
+  }, [apiKey, apiKeyErrorMessage, makeStatsigApiRequest]);
+
+  return (
+    <Stack flexDirection="column" alignItems="start" margin="spacing2Xl" style={{width: '700px'}}>
+      <Heading>Statsig App Configuration</Heading>
+      <Text>
+        The Statsig app provides a new Content Type (Statsig variant container) that allows
+        for automatic creation of experiments with different treaments directly in Statsig.
+        Results for these experiments can be found in the link under the Statsig tab of the
+        respective Statsig variant container.
+      </Text>
+      <Box style={{width: '100%'}}>
+        <ExternalStatsigLink
+          variant='neutral'
+          url='https://docs.statsig.com/guides/contentful'
+          linkLabel='Go to Statsig Docs'
+        >
+          Learn more about how to setup and use this app by heading to the link below.
+        </ExternalStatsigLink>
+      </Box>
+      <FormControl style={{width: '100%'}}>
+        <FormControl.Label isRequired>Statsig Console API Key</FormControl.Label>
+        <TextInput
+          data-testid="api-key"
+          value={apiKey}
+          onChange={handleApiKeyChanged}
+          onBlur={handleApiKeyBlurred}
+          isRequired={true}
+          isInvalid={!!apiKeyErrorMessage}
+        />
+        {!!apiKeyErrorMessage && (
+          <FormControl.ValidationMessage>{apiKeyErrorMessage}</FormControl.ValidationMessage>
+        )}
+      </FormControl>
+    </Stack>
+  );
+};
+
+
+const createVariantContainerContentType = async (sdk: ConfigAppSDK) => {
+  // noinspection JSDeprecatedSymbols
+  const variantContainer = await sdk.space.createContentType({
+    sys: {
+      id: VARIANT_CONTAINER_ID,
+    },
+    name: 'Statsig variant container',
+    description: 'Statsig variant container',
+    displayField: 'entryName',
+    fields: [
+      {
+        id: 'experimentId',
+        name: 'Statsig Experiment Id',
+        type: 'Symbol',
+        disabled: true,
+        required: false,
+        localized: false,
+        omitted: true,
+      },
+      {
+        id: 'entryName',
+        name: 'Entry Name',
+        type: 'Symbol',
+        required: true,
+        localized: false,
+      },
+      {
+        id: 'controlVariation',
+        name: 'Default Variation (control)',
+        type: 'Link',
+        localized: false,
+        required: true,
+        validations: [],
+        disabled: false,
+        omitted: false,
+        linkType: 'Entry',
+      },
+      {
+        id: 'treatmentVariations',
+        name: 'Treatment Variations',
+        type: 'Array',
+        localized: false,
+        required: true,
+        validations: [{ size: { min: 1, max: 10 } }],
+        disabled: false,
+        omitted: false,
+        items: {
+          type: 'Link',
+          linkType: 'Entry',
+        },
+      },
+    ],
+  });
+  // noinspection JSDeprecatedSymbols
+  await sdk.space.updateContentType(variantContainer);
```

Comment:
I noticed you're using a deprecated SDK method here. Have you look at using the newer `cma.contentType.update` method instead?  Let us know if you have any questions.

> @deprecated
> since version 4.0.0 Consider using cma.contentType.update instead See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details